src/pages/jaehyun/Main/JaehyunMain.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import './JaehyunMain.scss';
+import Nav from '../../../components/Nav/Nav';
+import Content from './Content/Content';
+
+class JaehyunMain extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userName: [],
+      newContent: '',
+      contents: [],
+    };
+  }
+
+  currentState = e => {
+    const newContent = e.target.value;
+    this.setState({ newContent: newContent });
```

Comment:
```suggestion
    this.setState({ newContent });
```
객체 단축 속성명을 이용할 수 있습니다.